changelog.d/20240910_134547_dmitrii.lavrukhin_fix_skeleton_project_export.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Fixing a problem with exporting a project with skeleton tracks
```

Comment:
Can you be more specific about what the problem was?

Reply:
tried to be more specific